src/common/questionnaire.json

```diff
@@ -5,134 +5,134 @@
+        { "optionId": 7, "optionLabel": "Tongue scraper", "points": 2 }
       ]
     },
     {
       "id": 4,
       "title": "Do you grind or clench your teeth?",
       "type": "radio",
       "options": [
-        { "optionId": 1, "optionLabel": "Yes" },
-        { "optionId": 2, "optionLabel": "No" },
-        { "optionId": 3, "optionLabel": "Not sure" }
+        { "optionId": 1, "optionLabel": "Yes", "points": 1 },
+        { "optionId": 2, "optionLabel": "No", "points": 4 },
+        { "optionId": 3, "optionLabel": "Not sure", "points": 0 }
       ]
     },
     {
       "id": 5,
       "title": "How often do your gums bleed when flossing?",
-      "type": "dropdown",
+      "type": "radio",
       "options": [
-        { "optionId": 1, "optionLabel": "Never" },
-        { "optionId": 2, "optionLabel": "Occasionally" },
-        { "optionId": 3, "optionLabel": "Frequently" },
-        { "optionId": 4, "optionLabel": "Always" }
+        { "optionId": 1, "optionLabel": "Never", "points": 4 },
+        { "optionId": 2, "optionLabel": "Occasionally", "points": 2 },
+        { "optionId": 3, "optionLabel": "Frequently", "points": 1 },
+        { "optionId": 4, "optionLabel": "Always", "points": 0 }
       ]
     },
     {
       "id": 6,
       "title": "How many acidic foods or beverages do you consume in a day? (e.g., citrus fruits, carbonated beverages, wine, etc.)",
       "type": "range",
       "options": [
-        { "optionId": 1, "optionLabel": "None" },
-        { "optionId": 2, "optionLabel": "1-2" },
-        { "optionId": 3, "optionLabel": "3-5" },
-        { "optionId": 4, "optionLabel": "5 or more" }
+        { "optionId": 1, "optionLabel": "None", "points": 4 },
+        { "optionId": 2, "optionLabel": "1-2", "points": 3 },
+        { "optionId": 3, "optionLabel": "3-5", "points": 2 },
+        { "optionId": 4, "optionLabel": "5 or more", "points": 1 }
       ]
     },
     {
       "id": 7,
       "title": "Do you experience tooth sensitivity to hot or cold?",
       "type": "radio",
       "options": [
-        { "optionId": 1, "optionLabel": "Yes" },
-        { "optionId": 2, "optionLabel": "Occasionally" },
-        { "optionId": 3, "optionLabel": "Never" }
+        { "optionId": 1, "optionLabel": "Yes", "points": 0 },
+        { "optionId": 2, "optionLabel": "Occasionally", "points": 2 },
+        { "optionId": 3, "optionLabel": "Never", "points": 4 }
       ]
     },
     {
       "id": 8,
       "title": "If you experience TMJ (temporomandibular joint) pain or discomfort, how would you rate your pain?",
       "type": "range",
       "options": [
-        { "optionId": 1, "optionLabel": "0" },
-        { "optionId": 2, "optionLabel": "1-2" },
-        { "optionId": 3, "optionLabel": "3-4" },
-        { "optionId": 4, "optionLabel": "5-6" },
-        { "optionId": 5, "optionLabel": "7-8" },
-        { "optionId": 6, "optionLabel": "9-10" }
+        { "optionId": 1, "optionLabel": "0", "points": 4 },
```

Comment:
for 6 options, the maximum points would be 6.

So, if each question is worth 10 points, then each point in a question would be worth `10/6` points.